clinvar/src/it/test-files/outputs/clinical_assertion/part-00000-of-00005.json

```diff
@@ -0,0 +1,3 @@
+{"content":"{\"ClinVarAccession\":{\"@Type\":\"SCV\"}}","id":"SCV000024774","version":3,"variation_id":"4352","vcv_id":"VCV000004352","submitter_id":"3","submission_id":"3.2013-12-16","internal_id":"24774","assertion_type":"variation to disease","date_created":"2011-01-25","date_last_updated":"2019-05-02","record_status":"current","review_status":"no assertion criteria provided","title":"ATL1, ARG415TRP_SPASTIC PARAPLEGIA 3, AUTOSOMAL DOMINANT","local_key":"606439.0007_SPASTIC PARAPLEGIA 3, AUTOSOMAL DOMINANT","interpretation_description":"Pathogenic","interpretation_last_evaluated":"2013-06-01","interpretation_comments":[],"clinical_assertion_trait_set_id":"SCV000024774","clinical_assertion_observation_ids":["SCV000024774.1"],"trait_set_id":"1198","rcv_accession_id":"RCV000004600"}
+{"content":"{\"ClinVarAccession\":{\"@Type\":\"SCV\"},\"Interpretation\":{\"Citation\":[{\"@Type\":\"general\",\"ID\":{\"@Source\":\"PubMed\",\"$\":\"28615637\"}},{\"ID\":{\"@Source\":\"PubMed\",\"$\":\"3344762\"}},{\"ID\":{\"@Source\":\"PubMed\",\"$\":\"21965116\"}}]},\"Citation\":[{\"@Type\":\"general\",\"ID\":{\"@Source\":\"PubMed\",\"$\":\"28615637\"}},{\"@Type\":\"general\",\"ID\":{\"@Source\":\"PubMed\",\"$\":\"28615637\"}}],\"Comment\":{\"@Type\":\"public\",\"$\":\"The mutation was identified with targeted resequencing using Ion Ampliseq Custom Panel and Ion PGM Sequencer.\"}}","id":"SCV000328419","version":1,"variation_id":"267450","vcv_id":"VCV000267450","submitter_id":"505727","submission_id":"505727.2016-10-25","internal_id":"623989","assertion_type":"variation to disease","date_created":"2016-11-03","date_last_updated":"2019-03-31","record_status":"current","review_status":"no assertion criteria provided","local_key":"NM_020964.2:exon 15;NM_020964.2:exon 17-22|OMIM:242840","submitted_assembly":"GRCh37","interpretation_description":"Pathogenic","interpretation_last_evaluated":"2016-08-11","interpretation_comments":["{\"text\":\"Patient, a 2 year-old girl, showed severe developmental delay,hypotonia, hypopigmentation, and high-arched palate. Her last head circumference was 42.5 (-3.4SD). This mutation was confirmed compound geterozygosity.\",\"type\":\"public\"}"],"clinical_assertion_trait_set_id":"SCV000328419","clinical_assertion_observation_ids":["SCV000328419.1"],"trait_set_id":"9271","rcv_accession_id":"RCV000496984"}
```

Comment:
@danxmoran is it important to keep the old filenames for these test outputs?

Reply:
No, it's fine for them to be renamed